src/woo_publications/conf/base.py

```diff
@@ -445,33 +88,6 @@
     '<div class="content">'  # note that this only applies to the admin
 )
 
-#
-# SENTRY - error monitoring
-#
-SENTRY_DSN = config("SENTRY_DSN", None)
-
-if SENTRY_DSN:
-    SENTRY_CONFIG = {
-        "dsn": SENTRY_DSN,
-        "release": RELEASE,
-        "environment": ENVIRONMENT,
-    }
-
-    sentry_sdk.init(
-        **SENTRY_CONFIG, integrations=get_sentry_integrations(), send_default_pii=True
-    )
```

Comment:
I think this breaks the integration to include a public sentry DSN for the frontend? but I'm not sure